packages/react-utilities/src/compose/types.ts

```diff
@@ -113,6 +113,34 @@ export type ComponentState<Shorthands extends ObjectShorthandPropsRecord> = {
   };
 } & Shorthands;
 
+/**
+ * This is part of a hack to infer the element type from a native element *props* type.
+ * The only place the original element is found in a native props type (at least that's workable
+ * for inference) is in the event handlers, so some of the helper types use this event handler
+ * name to infer the original element type.
+ *
+ * Notes:
+ * - Using an extremely obscure event handler reduces the likelihood that its signature will be
+ *   modified in any component's props.
+ * - Inferring based on a single prop name instead of a larger type like `DOMAttributes<T>` should be
+ *   less expensive for typescript to evaluate and is less likely to result in type expansion in .d.ts.
+ */
+type SomeEventName = 'onLostPointerCaptureCapture';
```

Comment:
Because some() is a function in JS this name confused me a little.  How about ObscureEventName?